<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>muon_sach</class>
 <widget class="QMainWindow" name="muon_sach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 1. TỔNG QUAN: Font chữ và nền chính */
QWidget {
    background-color: #FFFFFF; /* Nền trắng tinh khôi */
    color: #2c3e50; /* Chữ màu xám đậm (không dùng đen xì) cho dịu mắt */
    font-family: &quot;Segoe UI&quot;, sans-serif;
}

/* Nền của cửa sổ chính có thể cho xám cực nhạt để làm nổi bật các ô nhập liệu */
QMainWindow {
    background-color: #f8f9fa; 
}

/* 2. Ô NHẬP LIỆU: Tròn trịa, sáng sủa */
QLineEdit, QSpinBox, QDateEdit, QComboBox {
    background-color: #FFFFFF;
    border: 1px solid #e0e0e0; /* Viền xám rất mờ */
    border-radius: 8px; /* Bo tròn nhiều hơn cho mềm mại */
    padding: 8px 12px; /* Khoảng cách chữ rộng rãi */
    selection-background-color: #3498db;
}

/* Khi bấm vào thì sáng lên màu xanh ngọc */
QLineEdit:focus, QSpinBox:focus, QDateEdit:focus, QComboBox:focus {
    border: 2px solid #5dade2; /* Màu xanh sáng */
    background-color: #fbfdff;
}

/* 3. NÚT BẤM: Tươi tắn */
QPushButton {
    background-color: #3498db; /* Xanh dương tươi (Bright Blue) */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #5dade2; /* Sáng hơn khi di chuột */
}

QPushButton:pressed {
    background-color: #2980b9; /* Tối hơn chút khi ấn */
    padding-top: 11px;
}

/* Nút phụ (ví dụ nút Hủy) cho màu xám nhạt */
QPushButton#btnCancel {
    background-color: #ecf0f1;
    color: #7f8c8d;
}
QPushButton#btnCancel:hover {
    background-color: #bdc3c7;
}

/* 4. BẢNG DỮ LIỆU (TABLE): Thoáng, không kẻ ô chằng chịt */
QTableWidget {
    background-color: #FFFFFF;
    border: 1px solid #ecf0f1;
    border-radius: 8px;
    gridline-color: transparent; /* Ẩn đường kẻ lưới dọc/ngang đi cho thoáng */
    selection-background-color: #ebf5fb; /* Màu nền khi chọn dòng (Xanh cực nhạt) */
    selection-color: #2980b9; /* Màu chữ khi chọn */
    outline: 0; /* Bỏ đường viền nét đứt khi chọn */
}

/* Tiêu đề bảng */
QHeaderView::section {
    background-color: #FFFFFF;
    color: #7f8c8d; /* Chữ tiêu đề màu xám */
    padding: 10px;
    border: none;
    border-bottom: 2px solid #ecf0f1; /* Chỉ kẻ gạch chân dưới tiêu đề */
    font-weight: bold;
    text-transform: uppercase; /* Chữ in hoa */
}

/* 5. GROUP BOX (Khung nhóm) */
QGroupBox {
    border: 1px solid #e0e0e0;
    border-radius: 8px;
    margin-top: 20px;
    font-weight: bold;
    color: #3498db; /* Tiêu đề group màu xanh */
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
    background-color: #f8f9fa; /* Nền tiêu đề trùng nền cửa sổ */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>382</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tạo phiếu mượn</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Độc giả (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="doc_gia">
       <property name="styleSheet">
        <string notr="true">
QComboBox {
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: white;
    color: #333;
}
QComboBox:hover {
    border-color: rgb(69, 104, 173);
}

QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}
QComboBox::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); 
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow:on {
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    color: #333;
    border-top-width: 0px; 
    margin-top: -1px;
    padding-top: 4px;
    padding-bottom: 4px;
}
QComboBox QAbstractItemView::item {
    padding: 8px 12px;
    background-color: transparent;
}
QComboBox QAbstractItemView::item:hover {
    background-color: #f0f5ff;
    color: rgb(60, 91, 149);
}
QComboBox QAbstractItemView::item:selected, 
QComboBox QAbstractItemView::item:hover {
    background-color: #ebf1fa; /* Màu nền xanh pha xám rất nhạt */
    color: rgb(69, 104, 173);  /* Chữ màu xanh chủ đạo của app bạn */
    border-radius: 6px;        /* Bo góc item cho mềm mại */
    font-weight: bold;         /* Chữ đậm lên để nhấn mạnh */
    border: none;              /* Xóa viền nếu có */
}</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertPolicy::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tên sách (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ten_sach">
       <property name="styleSheet">
        <string notr="true">
QComboBox {
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: white;
    color: #333;
}
QComboBox:hover {
    border-color: rgb(69, 104, 173);
}

QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}
QComboBox::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); 
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow:on {
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    color: #333;
    border-top-width: 0px; 
    margin-top: -1px;
    padding-top: 4px;
    padding-bottom: 4px;
}
QComboBox QAbstractItemView::item {
    padding: 8px 12px;
    background-color: transparent;
}
QComboBox QAbstractItemView::item:hover {
    background-color: #f0f5ff;
    color: rgb(60, 91, 149);
}
QComboBox QAbstractItemView::item:selected, 
QComboBox QAbstractItemView::item:hover {
    background-color: #ebf1fa; /* Màu nền xanh pha xám rất nhạt */
    color: rgb(69, 104, 173);  /* Chữ màu xanh chủ đạo của app bạn */
    border-radius: 6px;        /* Bo góc item cho mềm mại */
    font-weight: bold;         /* Chữ đậm lên để nhấn mạnh */
    border: none;              /* Xóa viền nếu có */
}QComboBox {
    background-color: #FFFFFF;
    border: 1px solid #D1D5DB; /* Viền xám nhẹ */
    border-radius: 6px;
    padding: 6px 10px;
    min-height: 25px;
}

QComboBox:focus {
    border: 2px solid #2563EB; /* Viền xanh đậm khi focus */
}

/* Chỉnh thanh cuộn trong danh sách xổ xuống cho đẹp (Nếu danh sách dài) */
QComboBox QAbstractItemView QScrollBar:vertical {
    border: none;
    background: #F3F4F6;
    width: 8px;
    border-radius: 4px;
}
QComboBox QAbstractItemView QScrollBar::handle:vertical {
    background: #C1C1C1;
    border-radius: 4px;
}</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertPolicy::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Bản sao (*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ban_sao">
       <property name="styleSheet">
        <string notr="true">
QComboBox {
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: white;
    color: #333;
}
QComboBox:hover {
    border-color: rgb(69, 104, 173);
}

QComboBox:on {
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}
QComboBox::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); 
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow:on {
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 1px solid #B0B0B0;
    border-radius: 8px;
    color: #333;
    border-top-width: 0px; 
    margin-top: -1px;
    padding-top: 4px;
    padding-bottom: 4px;
}
QComboBox QAbstractItemView::item {
    padding: 8px 12px;
    background-color: transparent;
}
QComboBox QAbstractItemView::item:hover {
    background-color: #f0f5ff;
    color: rgb(60, 91, 149);
}
QComboBox QAbstractItemView::item:selected, 
QComboBox QAbstractItemView::item:hover {
    background-color: #ebf1fa; /* Màu nền xanh pha xám rất nhạt */
    color: rgb(69, 104, 173);  /* Chữ màu xanh chủ đạo của app bạn */
    border-radius: 6px;        /* Bo góc item cho mềm mại */
    font-weight: bold;         /* Chữ đậm lên để nhấn mạnh */
    border: none;              /* Xóa viền nếu có */
}</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertPolicy::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ngày mượn (*)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ngày trả (*)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDateEdit" name="ngay_muon">
         <property name="styleSheet">
          <string notr="true">/* --- PHẦN 1: Ô NHẬP LIỆU (Giống ComboBox) --- */
QDateEdit {
    border: 2px solid transparent;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #F3F4F6; /* Nền xám nhạt */
    color: #333;
    selection-background-color: rgb(69, 104, 173);
}

/* Khi di chuột vào */
QDateEdit:hover {
    background-color: #E5E7EB;
}

/* Khi đang nhập liệu (Focus) */
QDateEdit:focus {
    background-color: white;
    border: 2px solid rgb(69, 104, 173); /* Viền xanh */
}

/* Nút mũi tên xổ xuống */
QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

/* Icon (Bạn có thể dùng icon lịch 'calendar.png' thay vì mũi tên cho đúng điệu) */
QDateEdit::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); /* Hoặc file icon lịch của bạn */
    width: 16px;
    height: 16px;
}

/* --- PHẦN 2: LỊCH XỔ XUỐNG (QCalendarWidget) --- */

/* Khung tổng thể của lịch */
QCalendarWidget QWidget {
    alternate-background-color: #f9f9f9; 
    background-color: white;
    border-bottom-right-radius: 8px;
    border-bottom-left-radius: 8px;
}

/* Thanh điều hướng phía trên (Tháng/Năm) */
QCalendarWidget QToolButton {
    color: #333;
    background-color: transparent;
    border: none;
    font-weight: bold;
    icon-size: 20px; /* Chỉnh kích thước mũi tên trái phải của lịch */
}
QCalendarWidget QToolButton:hover {
    background-color: #F3F4F6;
    border-radius: 4px;
}

/* Menu chọn tháng năm khi bấm vào tiêu đề */
QCalendarWidget QMenu {
    background-color: white;
    border: 1px solid #ccc;
}

/* Bảng ngày tháng (Spin view) */
QCalendarWidget QAbstractItemView:enabled {
    font-size: 13px;
    color: #333;
    background-color: white;
    selection-background-color: rgb(69, 104, 173); /* Màu nền ngày được chọn */
    selection-color: white; /* Màu chữ ngày được chọn */
    outline: 0; /* Bỏ đường viền nét đứt xấu xí */
}

/* Ngày trong tuần (T2, T3, T4...) */
QCalendarWidget QAbstractItemView:disabled {
    color: #888; /* Màu chữ xám cho ngày không thuộc tháng hiện tại */
}</string>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="timeSpec">
          <enum>Qt::TimeSpec::UTC</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="ngay_tra">
         <property name="styleSheet">
          <string notr="true">/* --- PHẦN 1: Ô NHẬP LIỆU (Giống ComboBox) --- */
QDateEdit {
    border: 2px solid transparent;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #F3F4F6; /* Nền xám nhạt */
    color: #333;
    selection-background-color: rgb(69, 104, 173);
}

/* Khi di chuột vào */
QDateEdit:hover {
    background-color: #E5E7EB;
}

/* Khi đang nhập liệu (Focus) */
QDateEdit:focus {
    background-color: white;
    border: 2px solid rgb(69, 104, 173); /* Viền xanh */
}

/* Nút mũi tên xổ xuống */
QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

/* Icon (Bạn có thể dùng icon lịch 'calendar.png' thay vì mũi tên cho đúng điệu) */
QDateEdit::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); /* Hoặc file icon lịch của bạn */
    width: 16px;
    height: 16px;
}

/* --- PHẦN 2: LỊCH XỔ XUỐNG (QCalendarWidget) --- */

/* Khung tổng thể của lịch */
QCalendarWidget QWidget {
    alternate-background-color: #f9f9f9; 
    background-color: white;
    border-bottom-right-radius: 8px;
    border-bottom-left-radius: 8px;
}

/* Thanh điều hướng phía trên (Tháng/Năm) */
QCalendarWidget QToolButton {
    color: #333;
    background-color: transparent;
    border: none;
    font-weight: bold;
    icon-size: 20px; /* Chỉnh kích thước mũi tên trái phải của lịch */
}
QCalendarWidget QToolButton:hover {
    background-color: #F3F4F6;
    border-radius: 4px;
}

/* Menu chọn tháng năm khi bấm vào tiêu đề */
QCalendarWidget QMenu {
    background-color: white;
    border: 1px solid #ccc;
}

/* Bảng ngày tháng (Spin view) */
QCalendarWidget QAbstractItemView:enabled {
    font-size: 13px;
    color: #333;
    background-color: white;
    selection-background-color: rgb(69, 104, 173); /* Màu nền ngày được chọn */
    selection-color: white; /* Màu chữ ngày được chọn */
    outline: 0; /* Bỏ đường viền nét đứt xấu xí */
}

/* Ngày trong tuần (T2, T3, T4...) */
QCalendarWidget QAbstractItemView:disabled {
    color: #888; /* Màu chữ xám cho ngày không thuộc tháng hiện tại */
}</string>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::Section::DaySection</enum>
         </property>
         <property name="displayFormat">
          <string>d/M/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="muon_sach_button">
         <property name="text">
          <string>Mượn sách</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
